packages/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -63,11 +61,11 @@ export const GridRow = forwardRef<HTMLDivElement, GridRowProps>(
         /** Update the styles of the grid row via a subscription to prevent re-renders */
         const interactionStyleSubscription = combineLatest([
           gridLayoutStateManager.interactionEvent$,
-          gridLayoutStateManager.gridLayout$,
           gridLayoutStateManager.runtimeSettings$,
+          gridLayoutStateManager.gridLayout$,
```

Comment:
I am wondering if we can remove it since it's not used? So far I just placed it at the end of the array :)